src/coreclr/System.Private.CoreLib/src/System/Diagnostics/StackFrameHelper.cs

```diff
@@ -152,7 +152,7 @@ internal void InitializeSourceInfo(int iSkip, bool fNeedFileInfo, Exception? exc
             }
         }
 
-        public virtual MethodBase? GetMethodBase(int i)
+        public MethodBase? GetMethodBase(int i)
```

Comment:
I think this class is used by the unmanaged reflection stack. We should double check that changing the type's vtable layout won't break anything.

Reply:
Thanks :)